xls/public/c_api.h

```diff
@@ -21,7 +21,6 @@
 #include "xls/public/c_api_dslx.h"
 #include "xls/public/c_api_format_preference.h"
 #include "xls/public/c_api_ir_builder.h"
-#include "xls/public/c_api_vast.h"
```

Comment:
Here I am unsure; nothing in the header needs this, but maybe it was meant to by `// IWYU pragma: export` ?